CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing to Common Voice
+
+🎉 First off, thanks for taking the time to contribute! This project would not be possible without people like you. 🎉
+
+## Code of Conduct
+By participating in this project, you're agreeing to uphold the [Mozilla Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
+
+## How Can I Contribute?
+
+### Non-code
+Help us add more sentences for other volunteers to read. See [issue 341](https://github.com/mozilla/voice-web/issues/341) for details.
+
+### Bug fixes and feature enhancements
+All of our current issues can be found here on GitHub. Anything with a [help wanted](https://github.com/mozilla/voice-web/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) tag is up for grabs.
+
+#### Requirements
+- [NodeJS](https://nodejs.org/en/) (v7 or higher)
+- [npm](https://www.npmjs.com/) (4 or higher)
+
+#### Local Development
+```
+npm install
```

Comment:
Let's add a step before all these that says something like:

```
Install the latest version of NodeJS and NPM: https://nodejs.org/en/download/
```

Reply:
I forgot we have another dependency, ffmpeg. Let's add that here too.
```
Install ffmpeg
```

Reply:
Can you install ffmpeg on the command line or do you have it download it separately?

Reply:
@mikehenrty If we list installation of NodeJS + NPM, I think we should split that into a setup/requirements subsection. For me, the main part of "local development" should be concerned with the actual code-compile-test-debug loop. `npm install` is part of that loop IMO, since it's useful to do whenever you get updates from someone else (or switch branches), but installing Node/npm is not.

Reply:
looking at this again, I think we already cover these requirements (and thereby implicitly their installation) in the "Requirements" section, so I think the "Local development" section doesn't need to list their installation again

Reply:
I could always move the sentence @mikehenrty proposed to the above section. As in, list the requirements then add something like `If you haven't already, install the latest version of NodeJS and NPM: https://nodejs.org/en/download/` but I agree that it may be redundant.

@nevik What's the right link to install ffmpeg? And is there a version requirement?

Reply:
moving the instruction sentence up one section sounds good, @carolstran 👍 

@mikehenrty, wasn't ffmpeg removed as a development dependency some time ago? from what I can see in the code, it's only mentioned in the Puppet config (therefore only used in deployment?)

@carolstran, since ffmpeg is mostly a library, it should be installed using the OS package manager if possible. However, we can describe that in more detail in the future if this remains a dependency (that people usually need for local development). For now, including a link to ffmpeg's download page at https://www.ffmpeg.org/download.html should be enough :)